python-avd/pyavd/_eos_designs/structured_config/connected_endpoints/monitor_sessions.py

```diff
@@ -39,7 +39,7 @@ def monitor_sessions(self: AvdStructuredConfigConnectedEndpointsProtocol) -> Non
             for session_config in session_configs_list[1:]:
                 merged_settings._deepmerge(session_config)
 
-            if merged_settings.session_settings.access_group:
+            if merged_settings.session_settings._get("access_group"):
```

Comment:
This is a behavioral change, where we loose the default access-group. Why do we need this?

Reply:
We should test that only setting a source_settings ACL will not trigger the error with the old code.

Reply:
you are correct. _get will avoid inserting the access-group object into the data, otherwise triggering the boolean check to be true in line 63. So your change is good.